user_docs/en/userGuide.t2t

```diff
@@ -1672,6 +1672,21 @@ For this reason the option is enabled by default, interrupting speech when scrol
 
 Disabling this option allows speech to be heard while simultaneously reading Braille.
 
+==== Show selection ====[BrailleSettingsShowSelection]
+: Default
+  Enabled
+: Options
+  Default (Enabled), Enabled, Disabled
+:
+
+This setting determines if selection indicator (dots 7 and 8) is shown in braille display.
+
+The option is enabled by default so selection indicator is shown.
+
+Selection indicator might be a distraction while reading. Disabling this option may improve readability.
+

```

Comment:
This should be a single paragraph, with new sentences on new lines

```suggestion
This setting determines if selection indicator (dots 7 and 8) is shown in braille display.
The option is enabled by default, so that the selection indicator is shown.
The selection indicator might be a distraction while reading.
Disabling this option may improve readability.
```